ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Node/NodePageVulnerabilities.tsx

```diff
@@ -36,7 +32,9 @@ export type NodePageVulnerabilitiesProps = {
 
 function NodePageVulnerabilities({ nodeId }: NodePageVulnerabilitiesProps) {
     const { searchFilter } = useURLSearch();
-    const querySearchFilter = parseWorkloadQuerySearchFilter(searchFilter);
+
+    // TODO - Need an equivalent function implementation for filter sanitization for Node CVEs
+    const querySearchFilter = searchFilter;
```

Comment:
Instead of splitting up the `parseWorkloadQuerySearchFilter` helper into specifics for Node and Platform -- work that might be throwaway work when advanced filters are ready -- I opted to just remove this helper function throughout all of the Node and Platform CVE pages for now.

When in place, this helper would strip off the "CVE Snoozed" filter and make demonstration of this PR more difficult.